src/main/kotlin/com/msg/gcms/domain/attendance/util/impl/ScheduleConverterImpl.kt

```diff
@@ -0,0 +1,36 @@
+package com.msg.gcms.domain.attendance.util.impl
+
+import com.msg.gcms.domain.attendance.domain.entity.Schedule
+import com.msg.gcms.domain.attendance.presentation.data.dto.ScheduleDto
+import com.msg.gcms.domain.attendance.presentation.data.dto.ScheduleDto.ScheduleListDto
+import com.msg.gcms.domain.attendance.presentation.data.request.CreateScheduleRequestDto
+import com.msg.gcms.domain.attendance.util.ScheduleConverter
+import com.msg.gcms.domain.club.domain.entity.Club
+import org.springframework.stereotype.Component
+
+@Component
+class ScheduleConverterImpl : ScheduleConverter {
+    override fun toEntity(club: Club, scheduleDto: ScheduleDto): Schedule = Schedule(
+        id = 0L,
+        name = scheduleDto.name,
+        date = scheduleDto.date,
+        period = scheduleDto.period,
+        club = club,
+    )
+
+    override fun toEntities(club: Club, scheduleDto: ScheduleListDto): List<Schedule> = scheduleDto.schedules.map {
+        toEntity(
+            club = club,
+            scheduleDto = it
+        ) }
```

Comment:
```suggestion
        )
    }
```
요기 혹시 줄바꿈 안한게 맞는걸까요?